spec/v1.0-draft/field_types.md

```diff
@@ -46,6 +46,50 @@ _DigestSet (object)_
 > -   `{"sha256": "abcd"}` does not match `{"sha256": "fedb", "sha512": "abcd"}`
 > -   `{"somecoolhash": "abcd"}` uses a non-predefined algorithm
 
+<a id="Reference"></a>
+_Reference (object)_
+
+> A size-efficient representation of any hashable artifact referenced by
+> another in-toto attestation.
+>
+> Schema:
+>
+>```jsonc
+> {
+>   "localName": "<NAME>",
```

Comment:
👋  Is this changing the existing `name` attribute to `localName` in v1.0? I can see how `name` is ambiguous but this poses some problems for npm where we've already set up some verification rules around the existing in-toto subject format.. 😢 

We ended up using the `name` attribute to encode a package-url (package name + version) and compare this against the installed package when verifying provenance attestations. 

Is there any chance we can keep this attribute as `name` not to break existing in-toto v0.1 statements even though it's somewhat ambiguous?

Very much in favour of this change as it would help us encode where to download npm packages!

Reply:
or wait I'm confused, doesn't look like this is changing the subject structure but adding a `Reference` type. Is this something you can then use in the predicate?

Reply:
Thanks for the feedback! Yes, `Reference` is only a type to be used in a predicate. We are considering a future in which we add reference-like fields into the `subject`, such as a URI, but want to ensure it's fully backwards compatible for exactly the reasons you mention.

Reply:
Ah yes makes sense!